lib_device_control/module_build_info

```diff
@@ -11,6 +11,7 @@ OPTIONAL_HEADERS +=
 EXPORT_INCLUDE_DIRS = api
 
 INCLUDE_DIRS = $(EXPORT_INCLUDE_DIRS) \
-               src
+               src \
+               host
```

Comment:
Is it necessary to include the host directory for header files?  Is there a header file in that directory is being used to build the device code?

Reply:
Yes, we have an XC based host side too: [device_access_i2c_xcore.xc](https://github.com/xmos/lib_device_control/blob/develop/lib_device_control/host/device_access_i2c_xcore.xc).

Reply:
Thanks for the answer.